dev_docs/tutorials/data/search.mdx

```diff
@@ -0,0 +1,496 @@
+---
+id: kibDevTutorialDataSearchAndSessions
+slug: /kibana-dev-docs/tutorials/data/search-and-sessions
+title: Kibana data.search Services
+summary: Kibana Search Services 
+date: 2021-02-10
+tags: ['kibana', 'onboarding', 'dev', 'tutorials', 'search', 'sessions', 'search-sessions']
+---
+
+## Search service
```

Comment:
if you leave this out, and use 2 hashs for the below headings, you'll get more navigation levels on the right nav.

<img width="1196" alt="Screen Shot 2021-02-25 at 9 05 11 AM" src="https://user-images.githubusercontent.com/16563603/109165843-07f95c80-774a-11eb-8615-e28327e6d3c2.png">

See you don't see the section for "error handling" and there is only one top level entry, so it seems a bit wasteful of that space.